go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/jtolds/monkit-hw v0.0.0-20190108155550-0f753668cf20
 	github.com/klauspost/cpuid v0.0.0-20180405133222-e7e905edc00e // indirect
 	github.com/klauspost/reedsolomon v0.0.0-20180704173009-925cb01d6510 // indirect
+	github.com/kylelemons/godebug v1.1.0 // indirect
```

Comment:
remember to clean this up

Reply:
I noticed that it's modified when we are regenerating protobufs. Maybe need dependency there?

Reply:
The file is marked as `+build ignore`, however when running a file it ignores that and still adds it to `go.mod` :/... I tried to fix this in https://github.com/storj/storj/pull/2207 however that confused tools even more.

Reply:
Why don't we add go.mod and go.sum to the .gitignore. Everything already pushed up will stay up and we can update the go.mod and go.sum when necessary with a `git add --force go.mod go.sum`

Reply:
If you add it to `.gitignore` it will still track changes to `go.mod`. You can use `git update-index --assume-unchanged`, but that is more annoying to use.